bundles/org.eclipse.passage.lic.equinox/src/org/eclipse/passage/lic/internal/equinox/RequirementsSupplierRegistry.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.internal.equinox;
+
+import java.util.function.Supplier;
+
+import org.eclipse.passage.lic.internal.api.registry.Registry;
+import org.eclipse.passage.lic.internal.api.registry.StringServiceId;
+import org.eclipse.passage.lic.internal.api.requirements.ResolvedRequirements;
+
+@SuppressWarnings("restriction")
+public final class RequirementsSupplierRegistry implements Supplier<Registry<StringServiceId, ResolvedRequirements>> {
+
+	@Override
+	public Registry<StringServiceId, ResolvedRequirements> get() {
+
+		return null;
```

Comment:
hmm. what this strange literal stands for?

Reply:
It's autogenerated!  Sorry! Wiped!